client-app/ui-kit/components/molecules/line-item/vc-line-item.vue

```diff
@@ -19,6 +19,7 @@
         v-if="selectable"
         v-model="isSelected"
         class="vc-line-item__checkbox"
+        :name="$t('common.labels.toggle_vendor_select')"
```

Comment:
May be select item?

Reply:
Prefer to keep the context of this checkbox. It is not obvious for customer with screen reader the meaning of `Select item` at a header and at every line item.